pkg/envoyfilters/envoyfilters.go

```diff
@@ -26,21 +38,56 @@ type ACLRule struct {
 	Type string `json:"type"`
 }
 
+func (r *ACLRule) actionProto() envoy_rbacv3.RBAC_Action {
+	switch r.Action {
+	case "DENY":
+		return envoy_rbacv3.RBAC_DENY
+	case "ALLOW":
+		return envoy_rbacv3.RBAC_ALLOW
+	default:
+		panic("unknown action")
```

Comment:
Can we avoid the `panic`s in this file? We returned errors before so we could give the error to the actuator and webhook to handle and log them correctly.